protocol/read-write-resource/read-method-allow.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Servers MUST return Allow for GET and HEAD
+
+  Background: Set up container
+    * def testContainer = rootTestContainer.createContainer()
+    * def container = testContainer.createContainer()  
+    * def exampleTurtle = karate.readAsString('../fixtures/example.ttl')
+    * def rdfResource = testContainer.createResource('.ttl', exampleTurtle, 'text/turtle');
+    * def expected = parse(exampleTurtle, 'text/turtle')
+    
+  Scenario: GET is supported for containers
```

Comment:
The 4 scenario titles appear to have been copied from another test but I think they should refer to the Allow header which is being tested

Reply:
Yeah, I can say "allowed" instead, that is more precise.